add handling to long_running_execution_command for exceptions (#39532)
demisto/content: Packs/ProofpointEmailSecurity/Integrations/ProofpointEmailSecurityEventCollector/README.md

## Commands
### proofpoint-es-get-last-run-results
***
Retrieves the results of a connection attempt to Proofpoint, indicating whether it was successful or failed and why. If event fetching has been initiated, this command provides the results of the most recent fetch attempt.

## Known Limitations

The API does not allow use of the same token for more than one session at the same time. If you need to open more than one simultaneous connection to receive the same type of data, additional token(s) must be requested.

## Troubleshooting

> If you seem to have delays between the events being fetched and the current time / not seeing new events in general, it's recommended to split the instances to have one instance per event type to speed up the ingestion speed.

Reviewer: ```suggestion
If there are ingestion delays or events are missing, it's recommended to configure seperate instances per event type.
```
